### ISSUE TITLE
citation information missing from TTL files - 3

### ISSUE THREAD

**Issue by paregorios** [isawnyu/pleiades-gazetteer#396]
## Example problem:

```bash
$ curl https://pleiades.stoa.org/places/872503244/turtle | grep -A1 'cito:'
@prefix cito: <http://purl.org/spar/cito/> .
@prefix dcterms: <http://purl.org/dc/terms/> .
--
    cito:citesForInformation [ ],
        [ ];
--
    cito:citesAsDataSource [ ];
    owl:sameAs <http://pleiades.stoa.org/places/872503244/osm-location-of-roman-theater>;
```

## I would have preferred to see:

```bash
$ curl https://pleiades.stoa.org/places/872503244/turtle | grep -A1 'cito:'
@prefix cito: <http://purl.org/spar/cito/> .
@prefix dcterms: <http://purl.org/dc/terms/> .
--
    cito:citesForInformation <http://theatrum.de/1160.html>,
        <https://en.wikipedia.org/wiki/Salona>;
--
    cito:citesAsDataSource <http://www.openstreetmap.org/browse/way/93062850>;
    owl:sameAs <http://pleiades.stoa.org/places/872503244/osm-location-of-roman-theater>;
```

## tl;dr

 - The object in any ```cito:``` triple must be the value of the "Access URI" field.
 - If the "access uri" field is empty, use the value in the "Bibliographic URI" field.
 - If both the "access uri" and the "bibliographic uri" fields are empty, then do not generate a triple for this reference at all.

**Comment by paregorios** [comment]
blocker for #359, #397

**Comment by alecpm** [comment]
I believe I've deployed a fix for this to staging, but the place you have referenced on the tickets doesn't exist on staging and I don't want to waste a ton of time trying to hunt one down or create one to test. So I'll leave it to @paregorios to test. The change is fairly simple and should fix #397 as well. See: https://github.com/isawnyu/pleiades-rdf/commit/7753bd486c643038d386bc97191450dd80ee3d1b

I think this should be relatively harmless to deploy to production.

**Comment by paregorios** [comment]
Clearly something has changed on staging, but I don't think we've quite cracked the code. Here's a test case (irrelevant bits omitted):

```
$ curl https://pleiades.stoa.org/places/140481/turtle | grep -A1 'cito:'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  7270  100  7270    0     0  16243      0 --:--:-- --:--:-- --:--:-- 16263
    cito:citesAsRelated [ ];
    cito:citesForInformation [ ],
        [ ],
```

compared to staging:

```
$ curl https://pleiades.jazkarta.com/places/140481/turtle | grep -A1 'cito:'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  7223  100  7223    0     0   9721      0 --:--:-- --:--:-- --:--:--  9721
```

In other words, the citation information is completely missing from staging now, instead of being fully restored. 

Looking at [the commit](https://github.com/isawnyu/pleiades-rdf/commit/7753bd486c643038d386bc97191450dd80ee3d1b), I think the problem is that we are still looking at the "identifier" field, but that is deprecated and now unused. [See sub "tl;dr" above](https://github.com/isawnyu/pleiades-gazetteer/issues/396#issue-435296681) for the fields that should be tested instead. These fields superseded "identifier" back in 2016 with [PleiadesEntity:6315f0e](https://github.com/isawnyu/PleiadesEntity/commit/6315f0e37369bee9e16f4303cad454e5d408a49d)

**Comment by alecpm** [comment]
Ah I see. Sorry about that. Should be a quick fix now that I know the `identifier` isn't playing the role I thought it was.

**Comment by alecpm** [comment]
I think it's fixed now:

https://pleiades.stoa.org/places/140481/turtle

**Comment by paregorios** [comment]
I'm surprised for this particular example not to see the following:

 - http://www.perseus.tufts.edu/hopper/text?doc=Perseus:text:1999.04.0006:entry=saint-just-pepiron
 - http://francia.ahlfeldt.se/places/20147
 - http://www.persee.fr/web/revues/home/prescript/article/galia_0016-4119_1969_num_27_1_2518

Is it possible that the code is only looking at "bibliographicURI" and not the series of fallbacks described above sub "tl;dr"?

**Comment by alecpm** [comment]
Sorry, I did something very dumb here and pasted in `alternate_uri` instead of `access_uri`.

**Comment by alecpm** [comment]
Should be fixed now.

**Comment by paregorios** [comment]
And great joy seized the people! Please merge to master and deploy to production.
